src/sql/workbench/parts/notebook/cellToggleMoreActions.ts

```diff
@@ -141,3 +144,39 @@ export class ClearCellOutputAction extends CellActionBase {
 	}
 
 }
+
+export class RunCellsAction extends CellActionBase {
+	constructor(id: string, label: string, private isAfter: boolean,
```

Comment:
Minor: I think once we put stuff on extra lines, we have 1 line per parameter. Can we do this for consistency?